### PR TITLE
tabulatehelper

### PR DESCRIPTION
**Pull request by kiwi0fruit**
https://pypi.io/packages/source/t/tabulatehelper/tabulatehelper-0.0.7.tar.gz
doesn't work for some reason...

### CODE REVIEW

**Comment by kiwi0fruit** [comment]
@conda-forge-admin @conda-forge/help-python please review.

File: recipes/tabulatehelper/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{% set name = "tabulatehelper" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (beckermr) [inline recipes/tabulatehelper/meta.yaml]: What is this about?

Comment (kiwi0fruit) [reply recipes/tabulatehelper/meta.yaml]: @beckermr I don't know either. The download link by standard url doesn't work. But should. Republishing package on pypi doesn't help. But pip installs correctly as it uses this very hashed url as I put in the recipe. Strange...

Comment (kiwi0fruit) [reply recipes/tabulatehelper/meta.yaml]: @beckermr  I'm not sure if this hash url is permanent but it's stable enough to build a conda package.

**Comment by kiwi0fruit** [comment]
@beckermr What about merging?

**Comment by beckermr** [comment]
We can merge this one. Thanks!

**Pull request merged**